frappe/website/doctype/page_view/page_view.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020, Frappe Technologies and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+import frappe
+from frappe.model.document import Document
+
+class PageView(Document):
+	pass
+
+
+@frappe.whitelist(allow_guest=True)
+def make_view_log(path, referrer=None, browser=None, version=None):
```

Comment:
I think you can find the browser details in `frappe.request`

Reply:
Yeah, I figured it would be better to let the browser handle the part. Should I move it to python ?

Reply:
I don't know how much it matters. Might be interesting to see what Google Analytics does

Reply:
Yeah, I checked out freshlytics, a foss tool. They rely on server side parsing